gh-106996: Amend the introduction to the turtle graphics documentation (#106997)
python/cpython: Doc/library/turtle.rst

Introduction
============

Turtle graphics is an implementation of `the popular geometric drawing tools
introduced in Logo <https://en.wikipedia.org/wiki/Turtle_
(robot)>`_, developed by Wally Feurzeig, Seymour Papert and Cynthia Solomon

> in 1967.

Reviewer: Fascinating, I never knew this!

In Python, turtle graphics provides a representation of a physical "turtle"
(a little robot with a pen) that draws on a sheet of paper on the floor.

> It's a very effective and well-proven way for learners to encounter

Reviewer: Feel free to ignore :)

> ‘Substitute “damn” every time you’re inclined to write “very”. Your editor will delete it and the writing will be just as it should be.’ William Allen White

https://www.theguardian.com/guardian-observer-style-guide-w

```suggestion
It's an effective and well-proven way for learners to encounter
```
